test/presenters/organisations/show_presenter_test.rb

```diff
@@ -80,48 +80,6 @@
     assert_equal expected, @show_presenter.parent_organisations
   end
 
-  it 'formats policies correctly' do
```

Comment:
I think this means we can also remove the example this test uses: https://github.com/alphagov/collections/blob/master/test/support/organisation_helper.rb#L237

Reply:
Yep, now looking there is a few helpers that need to be removed too. Good spot!

Reply:
Removed! 👍